Find a functionally equivalent code:android.os.StrictMode.decrementExpectedActivityCount:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -30,36 +30,38 @@
     // noticeably less responsive during orientation changes when activities are
     // being restarted.  Granted, it is only a problem when StrictMode is enabled
     // but it is annoying.
-    Runtime.getRuntime().gc();
+    System.gc();
+    System.runFinalization();
+    System.gc();
     long instances = VMDebug.countInstancesOfClass(klass, false);
     if (instances > limit) {
         Throwable tr = new InstanceCountViolation(klass, instances, limit);

